Lib/shutil.py

```diff
@@ -117,9 +228,13 @@ def copyfile(src, dst, *, follow_symlinks=True):
     if not follow_symlinks and os.path.islink(src):
         os.symlink(os.readlink(src), dst)
     else:
+        if os.name == 'nt':
```

Comment:
This cannot be used in `shutil.copyfile`. My previous comment explains why in detail. It can be used in `shutil.copy2`, but this makes separate `copyfile` and `copystat` calls inconsistent with `copy2`. I suggested a possible way to address this.